src/components/ListboxGrid/ListboxGrid.tsx

```diff
@@ -14,7 +14,7 @@ import ElementResizeListener from '../ElementResizeListener';
 import {
   setDefaultValues, balanceColumns, calculateColumns, calculateExpandPriority, mergeColumnsAndResources,
 } from './distribute-resources';
-import { FoldedListbox } from '../FoldedListbox';
+import { FoldedListbox } from '../FoldedListbox/FoldedListbox';
```

Comment:
It might be a good idea to rename `FoldedListbox` into `index` so we don't get 3 levels with `FoldedListbox`